ceph_deploy/util/help_formatters.py

```diff
@@ -0,0 +1,11 @@
+import argparse
+
+
+class ToggleRawTextHelpFormatter(argparse.HelpFormatter):
+    """Inspired by the SmartFormatter at
+       https://bitbucket.org/ruamel/std.argparse
+   """
+    def _split_lines(self, text, width):
+        if text.startswith('R|'):
```

Comment:
Yeah, I can do that.  I put a lot of the explanation in the original commit message too: https://github.com/trhoden/ceph-deploy/commit/4858f3a3b6f339004cfe55a104ce031884013fd6